### ISSUE TITLE
What license should this project have?

### ISSUE THREAD

**Issue by ethanmdavidson** [kwebio/kweb-template#1]
I wasn't paying attention when I made this repo and put the GPLv3 on it, instead of LGPLv3 like the other kweb repos. However, after thinking about it some more I think a different license makes more sense for this repo.

Since this repo doesn't contain any kweb code, and it's entire purpose is to be copied and built on, I think we want to use [the unlicense](https://choosealicense.com/licenses/unlicense/). I'm not a lawyer, but I think this license will allow this project to be altered and redistributed without restriction, while the kweb-core project itself would still be under LGPLv3.

**Comment by sanity** [comment]
Ah, that's a very good point - I agree that we should use a license that doesn't impose any restrictions on use - which means GPL and LGPL are inappropriate.

I believe the most common license in this category is the [Apache License](https://www.apache.org/licenses/LICENSE-2.0), is there a reason to prefer the unlicense over that one?  I don't think I'd heard of it before.

**Comment by ethanmdavidson** [comment]
I don't understand the finer points here, I only suggested the unlicense because it was recommended by https://choosealicense.com/ as the license to use when you want to completely opt out of copyright restrictions. [Their page on Apache v2](https://choosealicense.com/licenses/apache-2.0/) lists "State Changes" and "License and copyright notice" as conditions of this license, but their summary also says "Licensed works, modifications, and larger works may be distributed under different terms and without source code", so I'm not sure exactly how it would work in this case.

**Comment by sanity** [comment]
Ah, I see - sorry if I jumped the gun on adding a license.

My only concern with unlicense was that it seemed like it might not be very well known - and this could cause issues because companies that are super-strict might require that a lawyer review it.

But on doing some reading it looks like it's more common than I thought - so I've switched the LICENSE file to unlicense per your original suggestion.
